### PR TITLE
bump

### PR DESCRIPTION
**Pull request by ocefpaf**
re-render to activate Windows.

### CODE REVIEW

**Comment by emthompson-usgs** [comment]
Looks like we both did essentially the same thing simultaneously (see #51), only difference is I also created a new release to get the latest updates of the source code.

**Comment by ocefpaf** [comment]
> Looks like we both did essentially the same thing simultaneously (see #51), only difference is I also created a new release to get the latest updates of the source code.

Your PR is great but it looks like the new tag does not include enough information for the version. Do you know if the `M_PI` was fixed? If not we may need to upstream my patch here.

**Comment by emthompson-usgs** [comment]
I did not fix `M_PI`

**Comment by ocefpaf** [comment]
@emthompson-usgs good news are that the patch works and this builds on Windows. I have a few experiments I want to run before upstreaming them.

The bad news is that our workaround in openquake did not work here :-(

I'll remove the bad openquake packages and rebuild with a proper patch to the manifest file.

**Comment by emthompson-usgs** [comment]
Wow, that's great that the C code looks to be compiling. Amazing!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,10 +6,12 @@ package:
 
 source:
   url: https://github.com/usgs/groundmotion-processing/archive/v{{ version }}.tar.gz
-  sha256: 731acd60df803dad609f363d0730959f9e68f3aab7637a013cf0aba67a329be0
+  sha256: 266091c7c53c4d21d7e9e593b8cd0ad3eb7f533f0d5bfba2c9ca33ac1ab1f59f
+  patches:
+    - winpi.patch  # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:11]: @emthompson-usgs do you want to upstream this patch or do you want me to send a PR?

Comment (emthompson-usgs) [reply recipe/meta.yaml:11]: I'm happy to do it if I can figure out what to do. Do you mean upstream as in within this feedstock in the parallel PR or elsewhere?

**Comment by emthompson-usgs** [comment]
@ocefpaf I meant to say that we are now able to build the upstream package in Windows on PRs with Azure CI. I thought it would make sense to merge that update with the changes you made here. Also, there was a typo in my commit above (I meant "lint" not "link").

**Comment by emthompson-usgs** [comment]
The problem is pretty clearly with how I configured setuptools-scm for reporting the code version, but I'm not seeing where the problem is, and I'm not able to reproduce the error locally.

**Comment by emthompson-usgs** [comment]
Building locally with `pip install -e` only worked if I pinned pip to 18.0 when creating the conda virtual environment. I can't explain why. The different version of pip is the only significant difference I can find between the package versions in the conda builds and what I have in the venv that successfully built this locally so I thought it was worth a try. But my attempt at pinning doesn't seem to be taking because the logs show that conda is still using pip 20.3.3.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml]: That's odd :-/
Did you try switching off the solver? If that is really setuptools_scm related then it is probably a bug.

**Comment by ocefpaf** [comment]
> Building locally with `pip install -e` only worked if I pinned pip to 18.0 when creating the conda virtual environment. I can't explain why. The different version of pip is the only significant difference I can find between the package versions in the conda builds and what I have in the venv that successfully built this locally so I thought it was worth a try. But my attempt at pinning doesn't seem to be taking because the logs show that conda is still using pip 20.3.3.

Let me take a look upstream, something is amiss here since modern pip should be OK with setuptools_scm.

File: recipe/meta.yaml (hunk 1 above)
Comment (emthompson-usgs) [reply recipe/meta.yaml]: Okay, I'm trying to roll back and reproduce the error I was getting. The discussion I found online when I searched for the error indicated it was a bug in pip. I'll link to it once I can find it again. Wouldn't turning off the solving break non-specified dependencies?

Comment (emthompson-usgs) [reply recipe/meta.yaml]: Here's the log when I try to install with unpinned pip (v20.3.3). Also seems to be related to the use of cython. 
```
clang is installed on your system.
gcc is installed on your system.
CC is set to 'x86_64-apple-darwin13.4.0-clang'
CC is set to x86_64-apple-darwin13.4.0-clang already.
Obtaining file:///Users/emthompson/src/python/groundmotion-processing
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/emthompson/miniconda/envs/gmprocess/bin/python3.8 /Users/emthompson/miniconda/envs/gmprocess/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /var/folders/00/_0gc_srn5sx936prflc1ln3c58bjh1/T/tmp6aaz3r0l
       cwd: /Users/emthompson/src/python/groundmotion-processing
  Complete output (16 lines):
  Traceback (most recent call last):
    File "/Users/emthompson/miniconda/envs/gmprocess/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
      main()
    File "/Users/emthompson/miniconda/envs/gmprocess/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/Users/emthompson/miniconda/envs/gmprocess/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/private/var/folders/00/_0gc_srn5sx936prflc1ln3c58bjh1/T/pip-build-env-wo6m7d0l/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 149, in get_requires_for_build_wheel
      return self._get_build_requires(
    File "/private/var/folders/00/_0gc_srn5sx936prflc1ln3c58bjh1/T/pip-build-env-wo6m7d0l/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 130, in _get_build_requires
      self.run_setup()
    File "/private/var/folders/00/_0gc_srn5sx936prflc1ln3c58bjh1/T/pip-build-env-wo6m7d0l/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 145, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 4, in <module>
      from Cython.Distutils import build_ext
  ModuleNotFoundError: No module named 'Cython'
  ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/emthompson/miniconda/envs/gmprocess/bin/python3.8 /Users/emthompson/miniconda/envs/gmprocess/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /var/folders/00/_0gc_srn5sx936prflc1ln3c58bjh1/T/tmp6aaz3r0l Check the logs for full command output.
Failed to pip install this package. Exiting.
```

**Comment by ocefpaf** [comment]
@emthompson-usgs let's work on the new version now! It took a while and some tweaks to openquake.engine but this is passing!

**Comment by emthompson-usgs** [comment]
Okay, amazing! I'll go re-open the PR for the the most recent version.

**Comment by emthompson-usgs** [comment]
I believe that I applied the patch in this PR upstream so future versions won't need it.

File: recipe/meta.yaml (hunk 1 above)
Comment (emthompson-usgs) [reply recipe/meta.yaml:11]: This turned out to be because I didn't have cython  in pyproject.toml, which is new to me.

**Pull request merged**